client/src/components/Toast.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div></div>
+  </template>
+  
+  <script lang="ts">
+  import "mosha-vue-toastify/dist/style.css";
+  import { createToast } from "mosha-vue-toastify";
+  export default {
+    setup() {
+      const toast = (title, color = "#217dbb") => {
+        createToast(title, {
+          position: "top-right",
+          hideProgressBar: true,
+          toastBackgroundColor: color,
+          timeout: 8000,
+          onClose: () => {
```

Comment:
do we need to define this?  because we don't do any actions on Closing the toast